README.rst

```diff
@@ -0,0 +1,184 @@
+Mozilla Product Details for Django
+==================================
+
+|Travis| |PyPI|
```

Comment:
Heh. Me too. This was automatically converted via pandoc. Pretty sweet I must say :)